### PR TITLE
added pango

### PR DESCRIPTION
**Pull request by eneoli**

### CODE REVIEW

**Comment by eneoli** [comment]
Already added with #54

**Comment by eneoli** [comment]
...or not

File: flake.nix [hunk 1]
```diff
@@ -51,13 +51,17 @@
...
           pkgs.mkShell.override
             {
               stdenv = pkgs.stdenvAdapters.useMoldLinker pkgs.clangStdenv;
             }
             {
-              packages = [ toolchain ];
+              packages = [
+                toolchain
+                pango
+              ];
```
Comment (TornaxO7) [inline flake.nix:64]:
```suggestion
              packages = with pkgs; [
                pango
              ] ++ [ toolchain ];
```
this looks cleaner in my opinion and you don't need this `pango = pkgs.pango` line afterwards in the let-in block.

**Review by TornaxO7 — CHANGES_REQUESTED** [review]

File: flake.nix [hunk 2]
```diff
@@ -51,13 +51,16 @@
         default =
           let
             toolchain = pkgs.rust-bin.fromRustupToolchainFile ./rust-toolchain.toml;
+            pango = pkgs.pango;
```
Comment (TornaxO7) [inline flake.nix:54]:
```suggestion
```

we don't need this one anymore

**Review by TornaxO7 — CHANGES_REQUESTED** [review]

**Review by TornaxO7 — APPROVED** [review]

**Pull request merged**